### DISCUSSION TITLE
Add icon to sys_user reference fields to impersonate user

### DISCUSSION THREAD

**Discussion by ajcooper72** [arnoudkooi/SN-Utils#339] [Ideas]
Whilst there already is the /imp command available for quickly impersonating users, how about injecting a link against any sys_user reference (whether in a list view or record) which will impersonate the user in the field?

![image](https://user-images.githubusercontent.com/15643760/208121592-0886edeb-ed9f-4c77-93c2-bc28fe137716.png)

**Comment by arnoudkooi** [comment]
I see the benefit, but I'm hesitant. Via the slash commands, it's easy as well. Don't want to add links, icons, etc. at too many different places.
